themes/hugo-fresh/layouts/partials/hero-body.html

```diff
@@ -5,6 +5,8 @@
 {{- $button1Link := index $hero "button1link" }}
 {{- $button2Text := index $hero "button2text" }}
```

Comment:
button2text: still also shows "Now in Beta!" which is wrong?

Reply:
like the changes from the other PR did not register?

Reply:
i think the PR didnt register because the build failed :|

Reply:
ah I see